contrib/pg_stat_statements/sql/test_pg_stat_statements.sql

```diff
@@ -0,0 +1,21 @@
+-- simple test to check there is no warnings during jumbling of the query
+CREATE TABLE table_test_pg_stat_statements
```

Comment:
Queries in tests should be as simple as possible, but sufficient to detect an error.
We should drop table before creating it, because the table may exist. For example
```
-- start_ignore
DROP TABLE IF EXISTS t;
-- end_ignore

CREATE TABLE t (a int) DISTRIBUTED BY (a);
SELECT GROUPING(a) FROM t GROUP BY GROUPING SETS ((a));
DROP TABLE t;
```

now the test is passed before the changes in queryjumble.c

Reply:
test was updated to cover this comment